Index: src/pages/integral/LotteryPage.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/pages/integral/LotteryPage.js	(date 1531881274000)
+++ src/pages/integral/LotteryPage.js	(date 1531881296707)
@@ -11,7 +11,7 @@
 
     render() {
         return <div>
-
+            1111
         </div>
     }
 }
